utils/data.py

```diff
@@ -1,12 +1,36 @@
 import discord
 
-from utils import permissions
+from utils import permissions, default
 from discord.ext.commands import AutoShardedBot, DefaultHelpCommand
 
+config = default.get("config.json")
+
 
 class Bot(AutoShardedBot):
     def __init__(self, *args, prefix=None, **kwargs):
         super().__init__(*args, **kwargs)
+        self.prefix = prefix
+
+        # Check if user desires to have something other than online
+        if config.status_type.lower() == "idle":
+            status_type = discord.Status.idle
+        elif config.status_type.lower() in ("do not disturb", "dnd"):
+            status_type = discord.Status.dnd
+        else:
+            status_type = discord.Status.online
+
+        # Check if user desires to have a different type of activity
+        if config.activity_type.lower() == "listening":
```

Comment:
Now that I see it, maybe make **config.activity_type.lower()** to a variable to then compare, makes it much cleaner considering I convert it to .lower() constantly.

Reply:
Yes that's better.